.github/workflows/test_local_integration.yaml

```diff
@@ -75,82 +78,29 @@ jobs:
           pip install .[dev]
           playwright install
 
-      - uses: azure/setup-kubectl@v4.0.0
-        with:
-          version: v1.19.16
-
-      - name: Enable docker permissions for user
-        run: |
-          sudo docker ps
-          sudo usermod -aG docker $USER && newgrp docker
-
-          docker info
-          docker ps
-
-      - name: Get routing table for docker pods
-        run: |
-          ip route
-
-      - name: Initialize Nebari Cloud
-        run: |
-          mkdir -p local-deployment
-          cd local-deployment
-          nebari init local --project=thisisatest --domain github-actions.nebari.dev --auth-provider=password
-
-          # Need smaller profiles on Local Kind
-          sed -i -E 's/(cpu_guarantee):\s+[0-9\.]+/\1: 0.25/g' "nebari-config.yaml"
-          sed -i -E 's/(mem_guarantee):\s+[A-Za-z0-9\.]+/\1: 0.25G/g' "nebari-config.yaml"
```

Comment:
I've removed these lines, because the default config does not have any `{cpu,mem}_guarantee` lines. They are probably just an artifact.

Reply:
Those were not artifacts, and they are part of the override settings (as seen below for example):
```yaml
  - display_name: Small Instance
    description: Stable environment with 2 cpu / 8 GB ram
    default: true
    kubespawner_override:
      cpu_limit: 2
      cpu_guarantee: 1.5
      mem_limit: 8G
      mem_guarantee: 5G
      image: quansight/nebari-jupyterlab:latest
```
Though this is not needed anymore since now we rely on CIrun to manage the local runner instances instead of vanilla GH Actions.